js/Model/Store/Storage.js

```diff
@@ -0,0 +1,28 @@
+import Validator from '../Validator.js';
+
+class Storage {
+  /**
+   * @param {string} product
+   */
+  set setStorageProduct(product) {
+    localStorage.setItem('product', JSON.stringify(product));
+  }
+
+  get getStorageProduct() {
+    return JSON.parse(localStorage.getItem('product'));
+  }
+
+  isPassStorageValidation() {
+    return Validator.storage.isEmpty(this.getStorageProduct) ? false : true;
```

Comment:
논리 NOT 연산자 (!) 사용으로 가독성을 높여보는 것도 좋을 것 같아요

```suggestion
    return !Validator.storage.isEmpty(this.getStorageProduct);
```